parent/pom.xml

```
@@ -18,7 +18,7 @@
   </description>
 
   <properties>
-    <version.quarkus>3.15.0</version.quarkus>
+    <version.quarkus> 3.19.0</version.quarkus>
```

🔧  To adopt all transitive updates let's bump to latest patch:
```suggestion
    <version.quarkus>3.19.3</version.quarkus>
```